sdk/python/kfp/__init__.py

```diff
@@ -12,7 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '1.1.0'
+# `kfp` is a namespace package.
```

Comment:
Are you sure kfp is a namespace package?

I'm also not sure the following line is needed for a namespace package.